docs/user-guide/side-bar.md

```diff
@@ -1,94 +1,91 @@
-
-* The informations will be returned as HTML accordingly.
-
-<img src="../img/side-bar/html_info.jpg" class="ms-docimage"/>
+<img src="../img/side-bar/format_example.jpg" class="ms-docimage"/>
 
 !!! warning
     This global settings could be overwritten by a layer-specific configuration (see [Feature Info Form](layer-settings.md#feature-info-form)).
 
-Below the point coordinates there are three icons:
 
-* **More Info** <img src="../img/button/more_info_icon.jpg" class="ms-docbutton"/> shows a popup window displaying the point address.
+In addition to the features, the following information is provided by the *Identify Tool*: 
 
-<img src="../img/side-bar/more_info.jpg" class="ms-docimage"/>
+* The **point address**  trought the *More Info* button  <img src="../img/button/more_info_icon.jpg" class="ms-docbutton"/>
 
-* **Coordinates Editor** <img src="../img/button/coordinates_editor_icon.jpg" class="ms-docbutton"/> makes available a tool to perform queries using decimal or aeronautical coordinates (we will explain more in depth this tool in the next paragraph).
+<img src="../img/side-bar/more-info.jpg" class="ms-docimage"/>
 
-* **Highlight Features** <img src="../img/button/highlight_features_icon.jpg" class="ms-docbutton"/> highlights the resulting features.
+* The **coordinates** <img src="../img/button/coordinates_editor_icon.jpg" class="ms-docbutton"/> of the point
 
-<img src="../img/side-bar/highlight_features.jpg" class="ms-docimage"/>
+<img src="../img/side-bar/coordinate.jpg" class="ms-docimage"/>
 
-Using the <img src="../img/button/zoom-layer.jpg" class="ms-docbutton"/> button it is possible to zoom the highlighted feature.
+!!! note
+    The point coordinates are viewed in **decimal** or **areonautical** format. It is possible to change the format by the *setting* button <img src="../img/button/gear_icon.jpg" class="ms-docbutton" style="max-heigth:50px;"/> 
+
+* The **Highlight Features** <img src="../img/button/highlight_features_icon.jpg" class="ms-docbutton"/> highlights the resulting features. Using the <img src="../img/button/zoom-layer.jpg" class="ms-docbutton"/> button it is possible to zoom the highlighted feature.
+
+<img src="../img/side-bar/hightlight-point1.gif" class="ms-docimage"/>
 
 ### Using the Coordinates Editor
 
-You might want to search features starting from their coordinates. The **Coordinates Editor** helps you to have more precision than just clicking on the map, so you can query objects by directly typing in their latitude and longitude.
+ In order to search for features starting from their coordinates it is possible using the **Coordinate Editor**. The user might precisely query objects by entering their latitude and longitude.  
+
+The point coordinates are searched in **decimal** or **areonautical** format. It is possible to change the format by the *setting* button <img src="../img/button/gear_icon.jpg" class="ms-docbutton" style="max-heigth:50px;"/> 
```

Comment:
The coordinates can be in **decimal** or **areonautical** format depending on the user needs. It is possible to change the format by the *setting* button <img src="../img/button/gear_icon.jpg" class="ms-docbutton" style="max-heigth:50px;"/>